resources/js/components/EideasyWidget.vue

```diff
@@ -0,0 +1,47 @@
+<script>
+import '@eid-easy/eideasy-widget';
+
+export default {
+    props: {
+        docId: String,
+        baseUrl: String,
+        clientId: String,
+    },
+    data() {
+        return {
+            apiEndpoints: {
+                identityStart: () => this.baseUrl + '/api/identity/start',
+                identityFinish: () => this.baseUrl + '/api/identity/finish',
+            },
+            enabledMethods: {
+               signature: 'all',
```

Comment:
Let's use the `"return_method_configs": true` prop in the  `api/signatures/prepare-files-for-signing` call so that the api will give us the signing methods that are actually available for the current session. We can then take these methods and provide them as an array to the `enabledMethods.signature` prop here. E.g. see the row 13 in the code example at https://eideasy-widget.docs.eideasy.com/guide/#minimal-1

Reply:
The reason for this is that the widget is a _dumb_ component (does not do any automatic config fetching) and has to be told which signing method buttons it should render.

Reply:
Great suggestion, here the fixup: [4401b51fea99f4cddc1babe8168b402d3444f110]